guides/it/Hardware.md

```diff
@@ -9,37 +10,41 @@ The current standard laptop specs for new purchases are:
 
 **Engineers and UCD**
 - MacBook Pro (14 inch)
-- Apple M3 with 8-core CPU, 10-core GPU, 16-core Neural Engine
-- 16GB unified memory
+- Apple M4 with 10-core CPU, 10-core GPU, 16-core Neural Engine
+- 32GB unified memory
 - 512GB SSD storage
-
-Please note that you can only have 1 external monitor set up with this laptop. We will be changing to M3 Pro chips which allow 2 external monitors.
+- 14" standard display
 
 OR
 
-- ThinkPad X1 Carbon Gen 11 14 inch
+- ThinkPad X1 Carbon Gen 11 or 12 14 inch
 - Core i7-1355U Processor (E-cores up to 3.70 GHz P-cores up to 5.00 GHz)
-- 16GB of RAM
+- 16GB or 32GB of RAM
 - 512GB SSD
-- Integrated Intel® Iris® Xe Graphics
 - Windows or Linux
 
+(Gen, spec and processor can vary on the ThinkPads - we'll talk to you individual about options.)
```

Comment:
I assume this covers display type and resolution?

Reply:
Yes. Though if you'd like to correct my poor grammar please do!